### PR TITLE
Add snowflake and sqlserver integrations

### PR DESCRIPTION
**Pull request by hugobarauna**

### CODE REVIEW

File: src/data/integrations.js [hunk 1]
```diff
@@ -283,6 +304,21 @@ const integrations = [
...
     },
   },
+  {
+    id: "snowflake",
+    name: "Snowflake",
+    logo: snowflakeLogo,
+    category: "Data Warehouse",
+    headline: "Connect to Snowflake directly from Livebook",
+    description:
+      "Connect to your Snowflake using Livebook's Database Connection Smart cell. Use Livebook Secrets to store your AWS access key securely. Write and run queries directly from Livebook.",
```
Comment (josevalim) [inline src/data/integrations.js:314]: We should say somewhere something like "Data exchange happens via the high-performance Apache Arrow format". You may also link to this article: https://www.snowflake.com/blog/fetching-query-results-from-snowflake-just-got-a-lot-faster-with-apache-arrow/ :)

**Review by josevalim — APPROVED** [review]

File: src/data/integrations.js (hunk 1 above)
Comment (hugobarauna) [reply src/data/integrations.js:314]: @josevalim maybe like that?

CleanShot 2023-09-14 at 15 27 09

Comment (josevalim) [reply src/data/integrations.js:314]: LGTM, your call.

Comment (hugobarauna) [reply src/data/integrations.js]: done ✅

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**